pkg/imgpkg/cmd/push_test.go

```diff
@@ -218,15 +255,19 @@ func Cleanup(dirs ...string) {
 }
 
 func createBundleDir(loc, imagesYaml string) error {
-	if imagesYaml == "" {
-		imagesYaml = emptyImagesYaml
-	}
-
-	bundleDir := filepath.Join(loc, ".imgpkg")
+	bundleDir := filepath.Join(loc, bundle.ImgpkgDir)
```

Comment:
```suggestion
	bundleDir := filepath.Join(loc, ".imgpkg")
```
we have this as a string on purpose to ensure that if we change, unintentionally, the const we have tests failing.

Reply:
Ah yes, very good point.